src/Microsoft.Health.Fhir.SqlServer/Features/Storage/SqlServerFhirDataStore.cs

```diff
@@ -440,11 +425,56 @@ private static string RemoveTrailingZerosFromMillisecondsForAGivenDate(DateTimeO
             return formattedDate.Replace(milliseconds, trimmedMilliseconds, StringComparison.Ordinal);
         }
 
-        private static string RemoveVersionIdAndLastUpdatedFromMeta(ResourceWrapper resourceWrapper)
```

Comment:
>RemoveVersionIdAndLastUpdatedFromMeta [](http://example.com/codeflow?start=30&length=37)

This method creates 2 more large strings. So for any update memory footprint = (2 versions) + 2 extra string per version *2 = 6x.